CHANGES.rst

```diff
@@ -7,6 +7,7 @@ Unreleased
 - Drop support for `Django 3.0`
 
 - Added urlsafe token field.
+- Introduce context manager for FieldTracker state reset (GH-#491)
 
 4.1.1 (2020-12-01)
 ------------------
```

Comment:
Although it is has already been released, IMHO we could change to the release notes of 4.1.0 to have
```
**Breaking changes:**
- `FieldTracker` now marks fields as not changed after `refresh_from_db`
- `FieldTracker` now respects `update_fields` changed in overridden `save()`
  method
- `FieldTracker` now resets states after `pre_save()` and not anymore `save()` signals, possibly altering the behaviour of overriden `save()` methods

Reply:
Great idea! Added this mark.